### PR TITLE
Setting assisted-installer-agent as default image in assisted-service…

### PR DESCRIPTION
**Pull request by tsorya**
…-deployment script

### CODE REVIEW

File: tools/deploy_assisted_installer_configmap.py [hunk 1]
```diff
@@ -42,12 +42,13 @@ def main():
             print("Deploying {}".format(DST_FILE))
 
             versions = {"IMAGE_BUILDER": "installer-image-build",
-                        "AGENT_DOCKER_IMAGE": "agent",
                         "IGNITION_GENERATE_IMAGE": "assisted-ignition-generator",
+                        "KUBECONFIG_GENERATE_IMAGE": "ignition-manifests-and-kubeconfig-generate",
```
Comment (eranco74) [inline tools/deploy_assisted_installer_configmap.py:46]: @tsorya @yevgeny-shnaidman this seems a bit off.
Didn't we change the name?

Comment (yevgeny-shnaidman) [reply tools/deploy_assisted_installer_configmap.py:46]: we did. KUBECONFIG_GENERATE_IMAGE is absolete as well as ignition-manifests-and-kubeconfig-generate

Comment (yevgeny-shnaidman) [reply tools/deploy_assisted_installer_configmap.py:46]: @tsorya  can you please remove this line?

**Pull request merged**